django/db/backends/base/features.py

```diff
@@ -227,6 +227,12 @@ class BaseDatabaseFeatures(object):
     # Place FOR UPDATE right after FROM clause. Used on MSSQL.
     for_update_after_from = False
 
+    # Combinatorial flags
+    supports_select_union = True
```

Comment:
I agree, default on sounds more sensible to me since they're part of the SQL standard